### ISSUE TITLE
Checkout code

### ISSUE THREAD

**Issue by deepakbandelach** [mfreiholz/iF.SVNAdmin#116]
Hi

I could able to setup the latest version in ubuntu server with LDAP integration. I can see the repositories and list of users without any problems. 

I have a question, How do I checkout the source code from the repositories as a LDAP user?

Thanks
Deepak

**Comment by mfreiholz** [comment]
Hey. You need to setup Apache for LDAP authentication. iF.SVNAdmin only syncs your LDAP with SVN's authorization file.

You may want to checkout the documentation: http://httpd.apache.org/docs/2.0/mod/mod_auth_ldap.html

**Comment by deepakbandelach** [comment]
Thanks for the advice.
I hope once the SVN repositories are exposed via apache2 server it can be accessed by user to checkout code.
if my understanding is right, iF.SVNAdmin maintains the svnauth file and code checkout happens via apache2 mod.
Is there a way by which the users can access the repo link from the svnadmin login? Somewhere in browse repositories they can get a link of the code to checkout, just like Github gives a link to clone the code.
ThanksDeepak

**Comment by mfreiholz** [comment]
Yes, see docs: http://svnadmin.insanefactory.com/documentation/#show-links-to-external-browsers
